Packs/Doppel/Integrations/Doppel/Doppel.py

```diff
@@ -30,6 +32,34 @@
 }
 DOPPEL_ALERT = "Doppel Alert"
 DOPPEL_INCIDENT = "Doppel Incident"
+DEFAULT_RETRY_TOTAL = 3
+DEFAULT_RETRY_BACKOFF_FACTOR = 2
+DEFAULT_RETRY_STATUS_LIST = [429, 500, 502, 503, 504]
+
+class FixedRetry(Retry):
```

Comment:
Is this the recommended way of implementing Retries in XSOAR? If yes, please share the reference

Reply:
The core api call is made using the http_request method of xsoar. The same method has a parameter status_list_to_retry which can be used here. We shoud utilise the same instead of using seperate class

Ref: https://github.com/metron-labs/content/blob/master/Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py#L2593